src/main/scala/uk/gov/homeoffice/cirium/services/health/CiriumAppHealthSummary.scala

```diff
@@ -86,6 +87,7 @@ case class AppHealthCheck(
             s"Cirium has been unresponsive for ${millisSinceContact / 1000} seconds - within allowable threshold")
           true
         } else {
+          metricsService.errorCounterMetric(s"isWithinHealthThresholds-withinThreshold")
```

Comment:
This is outside a threshold too, but a different threshold, so could be renamed '...-outsideAcceptableLostConnectivityDuration'